c2cgeoportal/scripts/theme2fts.py

```diff
@@ -241,3 +242,7 @@ def _add_layer(self, layer, interface, role):
             self._add_fts(layer, interface, "add_layer", role)
 
         return fill
+
+    @staticmethod
+    def _unicode_str(string):
+        return unicode(string, 'utf-8') if isinstance(string, str) else string
```

Comment:
gettext return sometime a unicode and sometime a sting?
we mostly use `string.decode('utf-8')` than `unicode(string, 'utf-8')`

Reply:
yes, str OR unicode...
Ok for the function.